### DISCUSSION TITLE
Examples with data binding to Html input elements ?

### DISCUSSION THREAD

**Discussion by giannik** [handsontable/hyperformula#961] [API Questions]
Are there any examples showing databinding to form input elements and html tables?
For example a html page  for invoice header and details table showing how to bind input elements to the calculation engine and when user changes value in one  cell the dependendant cells are updated in the UI.
Thanks

**Comment by sequba** [comment]
@giannik Take a look at these two demos:
1. https://handsontable.github.io/hyperformula/guide/demo.html <br/> - demonstrates displaying data in a HTML table with cell formulas that depend on other cells
2. https://handsontable.github.io/hyperformula/guide/basic-operations.html#demo <br/> - demonstrates changing the cell contents based on the input

I hope it helps. If you have any other questions feel free to ask.
